app/models/user.rb

```diff
@@ -14,6 +14,31 @@ def fake_username
     user_info[0]
   end
 
+  def total_keyword_count
+    total = 0
+    dreams.each do |dream|
+      total += dream.keyword_count
+    end
+    total
+  end
+
+  def average_word_count
+    total = 0
+    dreams.each do |dream|
+      total += dream.text.split.count
+    end
+    average = total/dreams.count
```

Comment:
Useless assignment to variable - `average`.<br>Surrounding space missing for operator `/`.